app/components/issue-tile.js

```diff
@@ -0,0 +1,18 @@
+import Component from '@ember/component';
+import { computed } from '@ember/object';
+
+export default Component.extend({
+    classNameBindings: ['isPointless', 'isClosed', 'isHidden'],
+    classNames: ['issue-tile'],
+
+    issue: null,
+    maxPoints: null,
+
+    url: computed.alias('issue.url'),
+    points: computed.alias('issue.storyPoints'),
+    name: computed.alias('issue.key'),
+
+    isPointless: computed.empty('points'),
+    isClosed: computed.equal('issue.status.name','Closed'),
```

Comment:
Do we want to track whether the issue is in a status of closed or whether it's in the last column of the board? The latter allows the board to define what closed means in jira - meaning it could hold more than one status name.